Samples/macOS-Swift/Shared/AppDelegate.swift

```diff
@@ -5,6 +5,7 @@ import Sentry
 class AppDelegate: NSObject, NSApplicationDelegate {
 
     func applicationDidFinishLaunching(_ aNotification: Notification) {
+        print("ProcessInfo.processInfo.arguments: \(ProcessInfo.processInfo.arguments)")
```

Comment:
`m`: Looks like a left-over to me, if not necessary let's remove it to keep the code minimal